CMakeLists.txt

```diff
@@ -599,6 +599,19 @@ CONFIGURE_FILE(
 	IMMEDIATE @ONLY
 )
 
+# Test setup
+ENABLE_TESTING()
+LIST(APPEND CMAKE_CTEST_ARGUMENTS "--verbose")
+
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  URL https://github.com/google/googletest/archive/refs/tags/v1.14.0.zip
+)
```

Comment:
_And a thousand packagers cried out in pain._ We'd have to ship a copy for everyone that needs offline builds and that's a blocker for me — gtest is huge in comparison to the other two.

Reply:
It's taken from the `Get started quickly` and works.

But yes, we had to find a way to vendor this (huge) thing for offline builds.

Reply:
On the other hand, it appears to be available in all major package systems.

Reply:
It's of no use for most users, so we can just remove this part and rely on developers getting it themselves.